config/helpers.go

```diff
@@ -0,0 +1,75 @@
+package config
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"strings"
+)
+
+func splitChannelList(channels string) string {
```

Comment:
I know this works, I just don't know if the IRC client library will actually, you know, let me write a multiline string.